src/configure.py

```diff
@@ -315,8 +315,7 @@ def cuda_check_error( result ):
     cuda_check_error( cuda.cuInit(0) )
     cuda_check_error( cuda.cuDeviceGetCount(ctypes.byref(nGpus)) )
 
-    arch = ""
-    if nGpus.value > 1: print("WARNING: More than one GPU. Selecting the last GPU architecture.")
+    if nGpus.value > 1: print("WARNING: More than one GPU. Selecting the last GPU compute capability.")
```

Comment:
Change the warning message to "WARNING: More than one GPU --> select the compute capability of the last GPU."